.github/auto_assign.yml

```diff
@@ -4,13 +4,14 @@ addReviewers: true
 # Set to true to add assignees to pull requests
 addAssignees: false
 
-# A list of reviewers to be added to pull requests (GitHub user name)
+# A list of reviewers to be added to pull requests (GitHub user name) or Github Group Name
 reviewers:
   - reviewerA
+  - reviewerGroupA
```

Comment:
I imagine these need to be real users or groups? Is this a test? If so, does this PR need a review?

Reply:
If this is meant to be a github team then it needs to be in the form `<org>/<team_name>` such as `Dispatchhealth/engcore`. If it's meant to be an autoassign reviewer group, then that group needs to be defined in this file (see farther down in likes 23-28

Reply:
@Jrc356 Actually this is a template so having the placeholders makes more sense?

Reply:
good catch, this is the actual auto-assign config file for this current repo, agree with the syntax will update